config/master/manifests.jsonnet

```diff
@@ -27,6 +27,39 @@ local regions = {
   },
 };
 
+local awsnodecm = {
+  configMap: {
+    apiVersion: "v1",
+    kind: "ConfigMap",
+    metadata: {
+      name: "aws-k8s-cni-configmap",
+      namespace: "kube-system",
+      labels: {
+        "k8s-app": "aws-node-cm",
+      },
+    },
+    data: {
+      AWS_VPC_CNI_NODE_PORT_SUPPORT: "true",
+      AWS_VPC_K8S_CNI_CUSTOM_NETWORK_CFG: "false",
+      AWS_VPC_ENI_MTU: "9001",
+      AWS_VPC_K8S_CNI_EXTERNALSNAT: "false",
+      AWS_VPC_K8S_CNI_RANDOMIZESNAT: "prng",
+      WARM_ENI_TARGET: "1",
+      AWS_VPC_K8S_CNI_LOGLEVEL: "DEBUG",
+      AWS_VPC_K8S_CNI_LOG_FILE: "/host/var/log/aws-routed-eni/ipamd.log",
+      AWS_VPC_K8S_PLUGIN_LOG_FILE: "/var/log/aws-routed-eni/plugin.log",
+      AWS_VPC_K8S_PLUGIN_LOG_LEVEL: "DEBUG",
+      DISABLE_INTROSPECTION: "false",
+      DISABLE_METRICS: "false",
+      AWS_VPC_K8S_CNI_VETHPREFIX: "eni",
+      ADDITIONAL_ENI_TAGS: "{}",
+      AWS_VPC_K8S_CNI_CONFIGURE_RPFILTER: "false",
+      DISABLE_TCP_EARLY_DEMUX: "false",
```

Comment:
`DISABLE_TCP_EARLY_DEMUX` is only used by the initContainer, not IPAMD. It's probably better not to have it here, unless we decide to give the full config to the init container as well?

Reply:
I would think we only want users setting variables in one place. Do you see any downside to giving the full config map to the init container?

Reply:
@mikestef19 good point. What I was aiming with this PR is to avoid any manual intervention to avoid env overrides made by customer and to run ‘kubectl apply -f ./..yaml’ during upgrades

Reply:
Oh ya I totally agree with the premise of the PR, as do many customers https://github.com/aws/containers-roadmap/issues/865

Reply:
Got it. I don't see any downside in giving full configMap to init container as there is no sensitive information, would like to hear @mogren @jayanthvn thoughts on this ? Happy to amend my PR to include that.

Reply:
@nithu0115 Oh, I don't mind that either, but then we need to add
```
        "envFrom":
          "value": "1"	        - "configMapRef":
            "name": "aws-k8s-cni-configmap"
```
to the init-container as well, that's what I meant, line 168 in your config.